src/test/java/com/project/portfolio/springboot/onlinesellingandshopping/Test.java

```diff
@@ -0,0 +1,13 @@
+package com.project.portfolio.springboot.onlinesellingandshopping;
+
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Test {
```

Comment:
This needs to be a unit test, where you mock dependencies for the class under test, and test happy path for each method, as well as corner cases. Take a look at the example here https://www.geeksforgeeks.org/how-to-write-test-cases-in-java-application-using-mockito-and-junit/